test/v2/unit/recovery.ts

```diff
@@ -431,7 +430,7 @@ describe('Recovery:', function() {
         ignoreAddressTypes: ['p2wsh', 'p2shP2wsh']
       });
 
-      recovery.transactionHex.should.equal('0100000001edda677a92037140f01cacc1ac0e22391adb9bf4dbce1c4c218822128ac6fd5301000000fc0047304402204a6e7373fa43494d78f3013ea4a0ea0f73a002d48cc29bfe8816b696280c2b08022056a5ad97886b5add39bca9135d445f36c99928b3c83bb0fc613c2a4a00feb93401473044022048de910b7a969ed2729fd9b628377cfdd1d2aa93036694a21bc39271b2f5e30a0220460b9a53ad4166738403bf60628ee4336f7ac984ac2ae1dce1398b30e30226a3014c69522103944ef399da9b45f26407dc3d7abd3544ef6fe6c32236bd6fe39f0db339f267a1210281941b32001f6fd10140e7a8f9a8df881f28d96f172528143e275d96c951a0422103a8ad848ac08c1fc598a91cb0786d6b663084ed3e399072de7c15c397629f2c6f53aeffffffff01b01198000000000017a91423dd90d164d5dfd07ba41814142a5bff0ad5446f8700000000');
+      recovery.transactionHex.should.equal('0100000001edda677a92037140f01cacc1ac0e22391adb9bf4dbce1c4c218822128ac6fd5301000000fc004730440220217d906eb64a6d30f58aed32cb7b2257684e556e7b0a70e6f1cbe8022786acec022063502bcc65f8ce16ee149efcf4e88e5fe1b09c69473d250e61dfba0466e0f36b01473044022037b0b776c65f32817ff183f422b16aaa4373ecaa96e1bcdd9d1cec230af681d3022000b3903c217e94f13305a6fe2100a151b3a6a9ef3b2bd7dfe2a7e27a711ce6cb014c69522103944ef399da9b45f26407dc3d7abd3544ef6fe6c32236bd6fe39f0db339f267a1210281941b32001f6fd10140e7a8f9a8df881f28d96f172528143e275d96c951a0422103a8ad848ac08c1fc598a91cb0786d6b663084ed3e399072de7c15c397629f2c6f53aeffffffff01640d98000000000017a91423dd90d164d5dfd07ba41814142a5bff0ad5446f8700000000');
```

Comment:
I couldn't find the inputs to this transaction on a testnet dash explorer, so haven't been able to verify if this is a valid tx yet or not

Reply:
(this doesn't mean its invalid, it just means the original test vector was not based on real inputs)